src/startup/client.js

```diff
@@ -0,0 +1,7 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client'
+import { settings } from 'settings'
+
+export const Client = () => new ApolloClient({
```

Comment:
Does this function really need to be a React Component? :smile:

Reply:
rename to client()

Reply:
Done